extensions/CheckUser/CheckUser.hooks.php

```diff
@@ -39,7 +39,7 @@ public static function updateCheckUserData( RecentChange $rc ) {
 			'cuc_title'      => $rc_title,
 			'cuc_minor'      => $rc_minor,
 			'cuc_user'       => $rc_user,
-			'cuc_user_text'  => $rc_user_text,
+			// 'cuc_user_text'  => $user->isAnon() ? $rc_user_text : '', // SUS-3080 - this column is redundant
 			'cuc_actiontext' => $actionText,
 			'cuc_comment'    => $rc_comment,
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$rc_comment" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app